Documentation/FontEditor/UsingFontEditor.md

```diff
@@ -0,0 +1,123 @@
+
+- The ***middle section*** contains two parts: the upper part which holds the entire glyph content of the font, or lack thereof if you are making a new one. And the lower part which contains ***Metadata*** information such as name, family, weight, slope, presentation size, mean line, baseline, glyph spacing and if the font is either fixed-width or variable-width. You can still further tweak your font paramaters via the Metadata section. The ` Fixed width ` toggle located at the right side next to Glyph spacing is of special note. It applies to the entire font file. It does not only apply to a single glyph or group of glyphs. It is the differentiating factor that informs the system if the font is or is not fixed-width. Fixed-width fonts are mostly used for Terminals and for displaying program source codes.
+
+- The ***right section*** which displays both search input for unicode blocks and the unicode block selection list.
+Metadata and unicode blocks can be turned on or off via ` Menu -> View ` option. Unicode block filters only what to display contained within the range of the block. Basic Latin covers 000-007F, Latin-1 Supplement covers 0080-00FF, Latin Extended-A covers 0100-017F and so on and so forth. Global search for glyph is affected when a certain block is currently selected. Only by selecting ` Show All ` will the global glyph search work as expected. So make it a habit of confirming that Show All is active before searching for a glyph.
+
+### The status bar
+The status bar displays additional information describing what each menu entry and toolbar icon does. It identifies what is the current unicode value of the glyph the cursor is in. It shows the glyph's visual representation (if available), glyph description and glyph dimensions. Second only to the glyph editor window, the status bar is your next best friend on your path to becoming a font master.
```

Comment:
It identifies the unicode value of the glyph currently under the cursor.
glyph description -> description
glyph dimensions -> dimensions

Reply:
Updated as such.